migration-from-4x.md

```diff
@@ -18,13 +18,13 @@ This chapter describes the migration process and workarounds for customers using
 
 To serve TinyMCE 5 from the cloud, include this in your html page:
 ```js
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
```

Comment:
We don't explain no-api-key here.

Reply:
added:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/migration-from-4x/